src/app/pages/StakingPage/Features/DelegationList/index.tsx

```diff
@@ -111,6 +113,7 @@ export const DelegationList = memo((props: Props) => {
           : t('delegations.reclaimedAmount', 'Amount to reclaim'),
       id: 'amount',
       selector: 'amount',
+      width: '28ex',
```

Comment:
would add `right: true,`  to amount and fee cols

Reply:
Good improvement!